.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
```

Comment:
This is used as a caption to the "Bug report" title. I like the version of the default github template a lot more. I think the tone of this is a bit informal. The default github proposal is:

```suggestion
about: Create a report to help us improve
```